openupgrade_scripts/scripts/event/17.0.1.8/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,67 @@
+---Models in module 'event'---
+---Fields in module 'event'---
+event        / event.event              / _order                        : _order is now 'date_begin, id' ('date_begin')
+event        / event.event              / activity_user_id (many2one)   : not related anymore
+event        / event.event              / activity_user_id (many2one)   : now a function
+# NOTHING TO DO
+event        / event.event              / auto_confirm (boolean)        : DEL
+# NOTHING TO DO: Attendees are now created confirmed (state=open)
+event        / event.event              / badge_format (selection)      : NEW required, selection_keys: ['A4_french_fold', 'A6', 'four_per_sheet'], hasdefault: default
```

Comment:
Next time, please document this nothing to do, saying that the default value is the same as the previous behavior.